lib/xcode_summary/plugin.rb

```diff
@@ -44,6 +44,12 @@ class DangerXcodeSummary < Plugin
     # @return   [Boolean]
     attr_accessor :test_summary
 
+    # Defines a RequestSource object for html/markdown formatting method calls.
+    # Defaults to `github`.
+    # @param    [RequestSource] value
+    # @return   [RequestSource]
+    attr_accessor :repo_provider
```

Comment:
Already not. As I see from discussion @marcelofabri offered a new solution just to get rid of it.